components/support/webextensions/README.md

```diff
@@ -14,6 +14,22 @@ Use Gradle to download the library from [maven.mozilla.org](https://maven.mozill
 implementation "org.mozilla.components:support-webextensions:{latest-version}"
 ```
 
+## Facts
+
+This component emits the following [Facts](../../support/base/README.md#Facts):
+
+| Action      | Item                      | Extras            | Description                  |
+|-------------|---------------------------|-------------------|------------------------------|
+| Interaction | web_extension_initialized | `extensionExtras` | Web extension is initalized. |
+
```

Comment:
nit: should this be `web_extensions_initialized`?

Reply:
Fixed.